### DISCUSSION TITLE
Google Auth Redirect URL update

### DISCUSSION THREAD

**Discussion by sparthe123** [DefectDojo/django-DefectDojo#5273] [Q&A] [answered]
Hi,

I need help in Google auth redirect URL , I need that to change to mydomain.com instead of 127.0.0.1:8080/.../

**Comment by valentijnscholten** [comment, marked as answer]
Please explain in detail your setup/goal/problem, or come to Slack as mentioned in the issue template. 
EDIT: Answer in comments below

**Reply by sparthe123** [reply]
Application is hosted in remote server and pointed with domain name to access it. 
I enabled Google Authentication setup , Once I authenticate with google , instead of redirecting to https://mydomain.com/complete/google-oauth2/  it redirect to http://127.0.0.1:8080/complete/google-oauth2/

I need help to update the redirect url to my domain.com.

**Reply by valentijnscholten** [reply]
did you follow the docs? https://defectdojo.github.io/django-DefectDojo/integrations/social-authentication/

**Reply by sparthe123** [reply]
Yes, I followed this doc and updated the client and secret in setting file as given in that instruction. But there is no information given for redirect URL.

Note: I setup Apache2 as frontend in that server to proxy the request to 8080 port in that server.

**Reply by valentijnscholten** [reply]
ah, at first it looked like the docs contained instructions to setup the redirect url, but it's just about authorizing redirect urls.

**Reply by sparthe123** [reply]
yeah , it contains instructions to be updated in google credentials page, I need help in updating server config to redirect to mydomain.com.

**Reply by sparthe123** [reply]
it started working once I removed apache proxy and configured ssl in nginx container .

**Reply by gnustavo** [reply]
I'm having this exact same problem. But I can't remove my apache proxy because I already use it to proxy other web systems on the same host, and it's already using the default HTTP/S ports.

I'm trying to understand the code but I'm having a hard time since it's all foreign to me. I think it's very abstract, perhaps because it's using some kind of "social framework". Can you point me to some relevant files or documentation where I could understand how to configure a redirect_uri to Google Auth?
